apps/src/tests/Test1649/state.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import * as jotai from 'jotai';
+import { AllowedDetentsType, SheetOptions } from './types';
+
+export const sheetInitialOptions: SheetOptions = {
+  sheetAllowedDetents: [0.4, 0.6, 0.9],
+  // sheetAllowedDetents: [0.6],
+  // sheetAllowedDetents: 'fitToContents',
+  sheetLargestUndimmedDetent: 2,
+  sheetGrabberVisible: false,
+  sheetCornerRadius: 24,
+  sheetExpandsWhenScrolledToEdge: true,
+  sheetInitialDetent: 0,
+}
+
+export const allowedDetentsAtom = jotai.atom<AllowedDetentsType>(sheetInitialOptions.sheetAllowedDetents);
+// const allowedDetentsAtom = jotai.atom<AllowedDetentsType>([0.6]);
+// const allowedDetentsAtom =
+//   jotai.atom<NativeStackNavigationOptions['sheetAllowedDetents']>(
+//     'fitToContents',
+//   );
```

Comment:
I'd remove comments here

Reply:
The comments were there to make toggling between values easier. I'll leave the comments in the `sheetInitialOptions` object and remove rest of them.